MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>Not for now but you can use the new az_auth and AzureSentinel stuff to do all of this infuture.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
updated with to use msticpy azure sentinel API